src/lib/viewers/error/__tests__/PreviewErrorViewer-test.js

```diff
@@ -55,7 +55,8 @@ describe('lib/viewers/error/PreviewErrorViewer', () => {
             ['zip', 'zip'],
             ['tgz', 'tgz'],
             ['flv', 'flv'],
-            ['blah', undefined],
+            ['pdf', 'pdf'],
+            ['pptx', 'pptx'],
```

Comment:
is it still possible for `[blah, undefined]` to happen?

Reply:
It will no longer be possible. If 'blah' is the extension, it will be passed into box-ui-elements [getIcon](https://github.com/box/box-ui-elements/blob/1850977829e4888139462a6f65794b1cffc52a08/src/components/preview/previewIcons.ts#L183) function which in the case where it can not find it, it will return the default icon that was previous being used when iconExtension was passed in as undefined.